tests/Filecoin/AddressTests.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright © 2017-2019 Trust.
```

Comment:
sorry for nitpicking, could you please search to replace copyright year? for new added files, should be 2020 now
```suggestion
// Copyright © 2017-2020 Trust.
```

Reply:
Sorry, missed that when copying :)